boost-configure

```diff
@@ -0,0 +1,11 @@
+
```

Comment:
Use `"buildsystem": "simple"` instead. Some examples: https://github.com/search?l=JSON&q=org%3Aflathub+boost&type=Code

Reply:
Also `$FLATPAK_BUILDER_N_JOBS` holds number of jobs to use.